GenderPayGap.WebUI/Views/Errors/PinExpired.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿@model GenderPayGap.WebUI.ErrorHandling.PinExpiredException
+
+@{
+    ViewBag.Title = "ERROR: Your PIN has expired";
+}
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <div class="govuk-box-highlight">
+            <h1 class="bold-large">Your PIN has expired</h1>
+        </div>
+        <div class="subsection panel-indent">
+            <div class="bold-small">Get a replacement PIN</div>
+        </div>
+        <p></p>
```

Comment:
Is this an accident or does it change the styling?

Reply:
I copied the content from [Error.cshtml](https://github.com/cabinetoffice/gender-pay-gap/blob/main/GenderPayGap.WebUI/Views/Shared/Error.cshtml) as this was used for displaying the error.

I checked on Firefox, Chrome and smaller screens and I can't see a difference. I'll remove it